app/src/main/java/com/duckduckgo/app/browser/omnibar/OmnibarLayoutViewModel.kt

```diff
@@ -126,6 +130,16 @@ class OmnibarLayoutViewModel @Inject constructor(
         GLOBE,
     }
 
+    init {
```

Comment:
I’d move this to `onAttachedToWindow`. There are cases where you’ll lose this ViewState update just because the view is not yet attached.